docs/design-language.md

```diff
@@ -238,72 +312,68 @@ Selected and Active states for list elements should showcase similar behavior to
 
 ---
 
-## Panels & Panel Navigation
+## Typography
 
-### Panels
+Good typography plays a large role in creating not only an aesthetically pleasing app but a usable one. It is essential that style guidelines and semantics are followed as they ensure consistency across our suite of applications. For example, by keeping headings and body text at predefined sizes makes it easier for developers to implement. There is no guessing required as to which size an H1 should be. Users working with our applications will instantly know they are using a VertiGIS app simply but the way it looks (and behaves).
 
-Using Panels in our application affords the user a good deal of information all contained in an easy to view place. Most information can be placed within a panel. Panels are designed with a minimum width of 365px so that they work on smart phone screens as well as larger tablet and desktop displays. The user can resize the panels with the provided drag handles. For full information regarding panels, refer to **<a href="https://developers.geocortex.com/docs/web/configuration-layout-getting-started/#panels" target="_blank">the Developer Center.</a>**
+### Type Scale
 
-### Panel Navigation
+<div class="typography buttons-box">
+    <h1>h1. Heading (2rem)</h1>
+    <h2>h2. Heading (1.812rem)</h2>
```

Comment:
A suggestion to make type scale readable:
html = 62.5% (converts to 10px)
body = 1.6rem (standard size 16px)
basically 1rem means for us 10px. For example body1 is 1.6rem, body2 1.4rem etc. 

And it is still accessible.

Reply:
Yeah, I've been doing this for most of my career prior to VertiGIS. In fact Studio Web does this. However, after some discussions with Eric a while back, we are going to move away from this and follow the pattern from MUI.